packages/cozy-doctypes/README.md

```diff
@@ -6,7 +6,121 @@ This repository is meant to bring together methods and functions for doctypes th
 - [Bank accounts](./src/banking/BankAccount.js)
 - [Balance histories](./src/banking/BalanceHistory.js)
 
+## `Document`
+
 The [Document](./src/Document.js) API is useful in NodeJS contexts where you need to have global models, available across files.
+It can be used as a base for other document types (see [BankTransaction](https://github.com/cozy/cozy-libs/blob/master/packages/cozy-doctypes/src/banking/BankTransaction.js) for example).
 
 ⚠️ The Document API does not work well in contexts where you need to change the client in flight. For this, prefer to use
 directly the `cozy-client` APIs.
+
+### How to use it
+
+Let's say we have a `io.cozy.simpsons` doctype, because we are developing a konnector that retrieves some Simpsons data. We create a class `Simpson` that inherits from `Document` :
```

Comment:
In the following section, we'll see how to deal with a fictitious `io.cozy.simpsons` doctype (we are developing a konnector to retrieve all the Simpsons characters). First, let's create a class `Simpson` that inherits from `Document`: